x/crosschain/keeper/evm_deposit.go

```diff
@@ -145,7 +154,11 @@ func (k Keeper) HandleEVMDeposit(ctx sdk.Context, cctx *types.CrossChainTx) (boo
 				)
 			}
 		}
+
```

Comment:
Consider putting lines 105 - 157 in a block , `{ }` 
For readability to show, where the context is branched and committed back

Reply:
I don't think you should use block if it doesn't have reason in the workflow